addons/im_livechat/static/tests/embed/transcript_sender.test.js

```diff
@@ -33,6 +33,8 @@ test("send", async () => {
     triggerHotkey("Enter");
     await contains(".o-mail-Message-content", { text: "Hello World!" });
     await click(".o-mail-ChatWindow-command[title*='Close']");
+    await contains(".o-livechat-closeConfirmation-overlay");
+    await click("button", { text: "Leave conversation" });
```

Comment:
Why not use this class `o-livechat-closeConfirmation-leave` instead of `button`? Using a class is more robust than `button` tag.